scripts/qa_crowbarsetup.sh

```diff
@@ -3159,7 +3159,7 @@ function oncontroller_manila_generic_driver_setup()
     openstack role add --project manila-service --user admin admin
     export OS_TENANT_NAME='manila-service'
     openstack image create --file $service_image_name \
-        --disk-format qcow2 manila-service-image
```

Comment:
@bmwiedemann fixed in https://github.com/toabctl/automation/commit/fb33b93596dd6b338b667ee6a53cac30c907f638 . Thanks!